iis-wf/iis-wf-import/src/main/resources/eu/dnetlib/iis/wf/importer/infospace/oozie_app/workflow.xml

```diff
@@ -75,6 +75,11 @@
             <value>projectorg</value>
             <description>project-organization output name</description>
         </property>
+        
+        <property>
+            <name>output_report</name>
+            <description>main directory for storing partial reports</description>
```

Comment:
I'm not sure we should call it partial reports. It depends on the context, from the whole processing POV those results are partial but from this importer POV those are just reports.